components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -105,14 +107,16 @@ export class GithubApp extends Probot {
         });
 
         app.on('installation.created', async ctx => {
-            const authId: string = ctx.payload.installation.account.id;
+            const accountId: string = `${ctx.payload.installation.account.id}`;
+            const authId: string = accountId;
             const user = await this.userDB.findUserByIdentity({ authProviderId: this.env.githubAppAuthProviderId, authId });
             const userId = user ? user.id : undefined;
-            await this.appInstallationDB.recordNewInstallation("github", 'platform', ctx.payload.installation.id, userId, ctx.payload.sender.id);
+            await this.appInstallationDB.recordNewInstallation("github", 'platform', accountId, userId, `${ctx.payload.sender.id}`);
```

Comment:
⚠️ ctx.payload.installation.id vs. ctx.payload.installation.account.id

Reply:
same?

Reply:
@geropl, AFAIR this question was the only important thing to have a look at. otherwise we should merge it to get rid of the technical dept.

Reply:
Fixed.